load/load.py

```diff
@@ -0,0 +1,41 @@
+import os 
```

Comment:
Fantastic job! You got it working and used the correct duckdb features, awesome! 🙌